lib/pry/helpers/text.rb

```diff
@@ -39,6 +39,18 @@ module Text
         end
       end
 
+      # Apply `color` and optional boldness to `text`
+      #
+      # @param color [Symbol] Color selected from `COLORS`
+      # @param text [String]
+      # @param bold [nil, true, false]
+      # @return [String] Text
+      def colorize(color, text, bold = false)
+        text
+          .then { |it| bold ? bold(it) : it }
```

Comment:
Can you add tests to this? I believe this is breaking due to the override of the `bold` method by the `bold` argument.

Reply:
Yep, done. It's weird because the code was working well on my side.

I've simplified a few things, you can test with `puts "\e[1;34mPry\e[0m"` (bold blue) if you need